openmc/mgxs_library.py

```diff
@@ -603,7 +603,8 @@ def nu_fission(self, nu_fission):
         if np.sum(self._nu_fission) > 0.0:
             self._fissionable = True
 
-    def set_total_mgxs(self, total, nuclide='total', xs_type='macro'):
+    def set_total_mgxs(self, total, nuclide='total', xs_type='macro',
+                       subdomain='all'):
```

Comment:
Actually nvm, I see your comment on the main page for this PR. Sounds good to me.